### PR TITLE
Fixes #770: Adds git-brv

### LINKED ISSUE #700: Time to release 4.5.0
@hemanth @nicolaiskogheim @qw3rtman
It has been 44 commits and seven months since the latest release.
I will make a new release this weekend if everything is ok.

### LINKED ISSUE #770: _Forking_ some Osse/git-scripts
Hi,

I stumbled upon this git script [git-brv](https://github.com/Osse/git-scripts/blob/master/git-brv), which I found quite interesting, I think this script will fit nicely in git-extras. 

* git-extras is installable via homebrew
* git-extras has possible more visibility and hence more maintainers

Of course proper credits must be given.
Screen Shot 2019-08-13 at 16 49 35

### PR DESCRIPTION
**Pull request by bric3**
This PR adds the git-brv script as proposed in #770.

This script requires bash 4.2, I don't know if this could be an issue.

### CODE REVIEW

File: man/git-brv.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+git-brv(1) -- Show commit logs since some date
```
Comment (spacewander) [inline man/git-brv.md]: Looks like the word is coming from `git-commits-since`? Remember to update the description here.

File: man/git-brv.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+========================================================
+
+## SYNOPSIS
+
+`git-brv` [&lt;date&gt;]
```
Comment (spacewander) [inline man/git-brv.md]: It seems the `date` argument is not supported by the script?

File: man/git-brv.md (hunk 1 above)
Comment (bric3) [reply man/git-brv.md]: Good catch ! yes indeed ;)

File: man/git-brv.md (hunk 2 above)
Comment (bric3) [reply man/git-brv.md]: Good catch as well thanks

File: man/git-brv.md (hunk 1 above)
Comment (bric3) [reply man/git-brv.md]: fixed

File: man/git-brv.md (hunk 2 above)
Comment (bric3) [reply man/git-brv.md]: fixed

**Comment by spacewander** [comment]
@bric3 
Merged. Thank you!

**Comment by bric3** [comment]
You are welcome! But I'm merely the messenger, big thanks to @Osse for his work !

**Pull request merged**